README.md

```diff
@@ -238,25 +238,36 @@ If you're working on our support code in `scripts/`, run `npm run fmt` to automa
-3. Determine the current version of the published stable documentation, e.g. at https://github.com/Qiskit/qiskit/releases
-4. Find a link to a CI artifact with the project's documentation, e.g at https://github.com/Qiskit/qiskit/suites/17881600359/artifacts/1026798160. To find this:
+1. Choose which documentation you want to generate (`qiskit`, `qiskit-ibm-provider`, or `qiskit-ibm-runtime`) and its version.
+2. Run `npm run gen-api -- -p <pkg-name> -v <version>`,
+   e.g. `npm run gen-api -- -p qiskit -v 0.45.0`
+
+If the version is not for the latest stable minor release series, then add `--historical` to the arguments. For example, use `--historical` if the latest stable release is 0.45.\* but you're generating docs for the patch release 0.44.3.
+
+## Generate new API docs
+
+1. Choose which documentation you want to generate (`qiskit`, `qiskit-ibm-provider`, or `qiskit-ibm-runtime`) and its version.
+2. Determine the current version of the published stable documentation, e.g. at https://github.com/Qiskit/qiskit/releases
+3. Download a CI artifact with the project's documentation. To find this:
    1. Pull up the CI runs for the stable commit that you want to build docs from. This should not be from a Pull Request
    2. Open up the "Details" for the relevant workflow.
       - Qiskit: "Documentation / Build (push)"
       - Runtime: "CI / Build documentation (push)"
       - Provider: "CI / Build documentation (push)"
    3. Click the "Summary" page at the top of the left navbar.
    4. Scroll down to "Artifacts" and look for the artifact related to documentation, such as `html_docs`.
-   5. Copy the link by right-clicking on the artifact.
-5. Run `npm run gen-api -- -p <pkg-name> -v <version> -a <artifact-url>`,
-   e.g. `npm run gen-api -- -p qiskit -v 0.45.0 -a https://github.com/Qiskit/qiskit/suites/17881600359/artifacts/1026798160`
-6. When opening your PR, include the CLI arguments you used. That helps us to know exactly how the docs have been generated over time.
-
-If the version is not for the latest stable minor release series, then add `--historical` to the arguments. For example, use `--historical` if the latest stable release is 0.45.\* but you're generating docs for the patch release 0.44.3.
+   5. Download the artifact by clicking on its name.
+4. Rename the downloaded zip file with its version number, e.g. `0.45.zip` for an artifact from `qiskit v0.45.2`.
+5. Upload the renamed zip file to https://ibm.ent.box.com/folder/246867649571
+6. Share the file by clicking the `Copy shared link` button
+7. Select `People with the link` and go to `Link Settings`.
+8. Under `Link Expiration` select `Disable Shared Link on` and set an expiration date of ~10 years into the future.
+9. Copy the direct link at the end of the `Shared Link Settings` tab.
+10. Modify the `scripts/lib/api/api-html-artifacts.json` file adding the new versions with the direct link from step 9.
+11. Run `npm run gen-api -- -p <pkg-name> -v <version>`,
```

Comment:
Needs to also mention `--historical` because of patch releases. But maybe instead have an `h3` like this:

```
### Generate patch releases

For example, if the current docs are for 0.45.2 but you want to generate 0.45.3.

Follow the same process above for new API docs, other than:

* When uploading the artifact, overwrite the existing file with the new one. Be careful to follow the above steps to change "Link Expiration"!
* Add `--historical` to the end of the `npm run gen-api` command.

Reply:
I added this new section, but without the last point that talks about the `--historical` argument. Generating patches of the current docs shouldn't be always historical. Instead, I added back the paragraph we had before at the end of the `Generate new API docs` section